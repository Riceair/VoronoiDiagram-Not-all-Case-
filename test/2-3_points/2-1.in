#雙點測試
2
289 290
342 541