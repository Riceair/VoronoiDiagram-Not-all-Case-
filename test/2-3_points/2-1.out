P 289 290
P 342 541
E -157814.5 33805.5 158445.5 -32974.5
